server/objects/settings/ServerSettings.js

```diff
@@ -97,6 +114,50 @@ class ServerSettings {
     this.logLevel = settings.logLevel || Logger.logLevel
     this.version = settings.version || null
 
+    this.authActiveAuthMethods = settings.authActiveAuthMethods || ['local']
+    this.authGoogleOauth20ClientID = settings.authGoogleOauth20ClientID || ''
+    this.authGoogleOauth20ClientSecret = settings.authGoogleOauth20ClientSecret || ''
+    this.authGoogleOauth20CallbackURL = settings.authGoogleOauth20CallbackURL || ''
+
+    this.authOpenIDIssuerURL = settings.authOpenIDIssuerURL || ''
+    this.authOpenIDAuthorizationURL = settings.authOpenIDAuthorizationURL || ''
+    this.authOpenIDTokenURL = settings.authOpenIDTokenURL || ''
+    this.authOpenIDUserInfoURL = settings.authOpenIDUserInfoURL || ''
+    this.authOpenIDClientID = settings.authOpenIDClientID || ''
+    this.authOpenIDClientSecret = settings.authOpenIDClientSecret || ''
+
```

Comment:
I would recommend adding an option to change the name of the OpenID button in the frontend. Most apps provide this functionality. Reason is, that most friends or family don't know what SSO/OpenID is, so one could name it just "Family Auth", "Click here" or something like that depending on preference